packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.css

```diff
@@ -1,3 +1,3 @@
 a {
-  color: var(--color-text-link-active, #0f2d5d);
+  color: var(--lwc-brandTextLinkActive, #014486);
```

Comment:
Why did the fallback value change?

Reply:
My bad I used a slightly different shade of blue, it's fixed now.